### ISSUE TITLE
version 5.0.3 has a breaking change with the post code

### ISSUE THREAD

**Issue by twisha16** [appnexus/anx-api#70]
This is how we are currently using it for login form
var api = new Api({
    target: api_base_url,
  });
  var authUrl = api_base_url + '/auth';
  return api
    .post(authUrl, {
      auth: {
        username: req.body.username,
        password: req.body.password,
      },
    })

It seems to be appending the host name twice in the uri now:
uri: 'http://01-thorondor-hbapi-test165605.envnxs.net/v1.18/http://01-thorondor-hbapi-test165605.envnxs.net/v1.18/auth' 
The auth fails due to that.

**Comment by joelgriffith** [comment]
There was a change in anx-api in how it builds URL's when supplying a target. You shouldn't need to build the fully-qualified path with this latest change now, so your code above should be:

```js
var api = new Api({
  target: api_base_url,
});

// No need to concatenate full api path
return api.post('/auth', {
  auth: {
    username: req.body.username,
    password: req.body.password,
  },
})
```

I think we should revert this change in the 5.x.x tag range, and do a major revision to signify that this is potentially breaking

**Comment by tsantef** [comment]
Fixed in v5.0.5
